examples/chart/teleport-agent/templates/deployment.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .Release.Name }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app: {{ .Release.Name }}
+spec:
+  replicas: {{ .Values.replicaCount }}
+  selector:
+    matchLabels:
+      app: {{ .Release.Name }}
+  template:
+    metadata:
+      labels:
+        app: {{ .Release.Name }}
+    spec:
+      containers:
+      - name: "teleport"
+        image: "{{ .Values.image }}:{{ .Values.teleportVersion }}"
+        args:
+        {{- if .Values.insecureSkipProxyTLSVerify }}
+        - "--insecure"
+        {{- end }}
+        volumeMounts:
+        - mountPath: /etc/teleport
+          name: "config"
+          readOnly: true
+        - mountPath: /var/lib/teleport
+          name: "data"
+      volumes:
+      - name: "config"
+        configMap:
+          name: {{ .Release.Name }}
+      - name: "data"
+        emptyDir: {}
```

Comment:
Just thinking out loud, nodes (and all our new `*_service`'s) are pretty much stateless, right? So it should be fine to use emptyDir.

Reply:
Our other charts also do this.

Reply:
they are stateless, except for their credentials (which can be re-generated on restart using a join token)